.github/workflows/nightly-eth-bench.yml

```diff
@@ -2,7 +2,7 @@ name: Nightly - ETH Bench
 
 on:
   schedule:
-    - cron: '0 1 * * *' # run at 1am to stagger tests
+    - cron: '0 1 * * *' # run at 1am to stagger tests (take around 15m to pass)
```

Comment:
1:30

Reply:
I guess the point was to avoid this running at the same time as ansible tests?